raft/confchange/confchange.go

```diff
@@ -265,7 +265,7 @@ func (c Changer) initProgress(cfg *tracker.Config, prs tracker.ProgressMap, id u
 		// making the first index the better choice).
 		Next:      c.LastIndex,
 		Match:     0,
-		Inflights: tracker.NewInflights(c.Tracker.MaxInflight),
+		Inflights: tracker.NewInflights(c.Tracker.MaxInflight, 0), // TODO: set maxBytes
```

Comment:
Not suggesting to do this here (really, not worth your time), but in the future, a neat trick could be to have a

```
const todoMaxBytes = 0
```

and reference that instead, that way it's easier to track down the usages later. Makes more sense with bigger changes of course.